avatars/avatars.js

```diff
@@ -3083,14 +1537,14 @@ class Avatar {
           const headPosition = eyePosition;
 
           // Look at direction in world coordinate
-          const lookAtDir = localVector2.copy(headPosition).sub(position).normalize();
-      
+          const lookAtDir = this.localVector2.copy(headPosition).sub(position).normalize();
```

Comment:
Why make all cache variables local to `this`?

Reply:
i made the local* variables avatar was using members of the avatar class, rather than defined outside of the class

Reply:
Yeah, IMO we do not need extras. If the implementation is polluting side effects across instances, that's probably a variable collision bug.

Otherwise this is only a waste of memory.